clang/lib/CodeGen/CGLoopInfo.cpp

```diff
@@ -882,3 +945,52 @@ void LoopInfoStack::InsertHelper(Instruction *I) const {
     return;
   }
 }
+
+void LoopInfo::CollectIVDepMetadata(
```

Comment:
I'm fine with the capital naming if other `LoopInfo` members follow it, but please check if this is the case and align if needed

Reply:
We are consistently inconsistent across the file, but most in LoopInfo are lower case, so I'll change to that.  Once it builds I'll update.